components/raftstore/src/store/fsm/apply.rs

```diff
@@ -525,7 +525,7 @@ where
             self.kv_wb().write_opt(&write_opts).unwrap_or_else(|e| {
                 panic!("failed to write to engine: {:?}", e);
             });
-            self.perf_context.report_metrics();
+            self.perf_context.report_metrics(&[]);
```

Comment:
Is the `&[]` a TODO?

Reply:
Yes, it's a TODO. But with `&[]`, it will work like it works now.

In https://github.com/tikv/tikv/pull/12783/files#diff-ee7d1a7f274d36b83db93153af39a61497eaea746565e713d02b91d68f25a180, you can see how it is finally passed in.